server/semantics-service/src/main/java/edu/illinois/ncsa/incore/service/semantics/utils/GeotoolsUtils.java

```diff
@@ -0,0 +1,119 @@
+package edu.illinois.ncsa.incore.service.semantics.utils;
+
+import org.apache.commons.io.FilenameUtils;
+import org.apache.log4j.Logger;
+import org.geotools.data.*;
+import org.geotools.data.shapefile.ShapefileDataStore;
+import org.geotools.data.shapefile.ShapefileDataStoreFactory;
+import org.geotools.data.simple.SimpleFeatureSource;
+import org.geotools.data.simple.SimpleFeatureStore;
+import org.geotools.feature.DefaultFeatureCollection;
+import org.geotools.feature.simple.SimpleFeatureTypeBuilder;
+import org.geotools.referencing.crs.DefaultGeographicCRS;
+import org.locationtech.jts.geom.Point;
+import org.opengis.feature.simple.SimpleFeatureType;
+
+import java.io.*;
+import java.util.*;
+
+public class GeotoolsUtils {
```

Comment:
Do you just decide to copy the same GeotoolsUtils to semantics service?
I think im ok with that but you might want to create subsequent task to put this and the fileUtil to incore-common

Reply:
Yes, I tried doing that but some of these fileutils and geotoolutils methods would stop working if I changed directories as it required some stuff from other files. Thats why I thought to just copy those functions to semantics

Reply:
I can create a task to document that.